pyvelox/pyvelox.h

```diff
@@ -327,17 +327,26 @@ inline void addVectorBindings(
       .value("DICTIONARY", velox::VectorEncoding::Simple::DICTIONARY)
       .value("FLAT", velox::VectorEncoding::Simple::FLAT)
       .value("SEQUENCE", velox::VectorEncoding::Simple::SEQUENCE)
-      .value("ROW", velox::VectorEncoding::Simple::ROW)
+      .value("ROW", velox::VectorEncoding::Simple::ROW, R"pbdoc(
+         Row vector encoding.
```

Comment:
ROW is not an encoding, but rather a type. Encodings are: flat, constant, dictionary. Types are: integer, bigint, array, map, row / struct, etc.

Reply:
I will update the wording.

Reply:
based on a review comment: https://github.com/facebookincubator/velox/pull/3363#discussion_r1054663016
We decided to remove this part for the moment, since there will be a follow up PR to write these documentation.